e2e/wdio/platform/pages/base-component.po.ts

```diff
@@ -19,6 +27,25 @@ export class BaseComponentPo {
         }
     }
 
+    saveExampleBaselineScreenshot(specName: string, areas: string = this.exampleAreaContainersArr, options: object = {}): void {
+        const areasArray = elementArray(areas);
+        for (let i = 0; i < areasArray.length; i++) {
+            scrollIntoView(areas, i);
+            saveElement(areas, `${specName}-example-${i}-platform-${getImageTagBrowserPlatform()}`, options, i);
```

Comment:
maybe better to do ${i + 1} so that there isn't 'example-0' and we can see it like 'example-1' is 1st the 1st example

Reply:
In this case, I need to recreate all images. Let's merge it as it is to unblock this methos for new components and then fix it if necessary